packages/back-end/src/controllers/organizations.ts

```diff
@@ -635,6 +649,14 @@ export async function deleteMember(
 
   await removeMember(org, id);
 
+  // The organization wasn't updating automatically, so I'm getting a fresh copy of the current organization.
```

Comment:
Not sure if there is a better way to do this. Getting the updatedOrg was the only way to ensure that we were getting the qty of members + invites AFTER those were updated.

Reply:
Updated!